Ports/thepowdertoy/patches/0001-malloc.h-doesn-t-exist-on-Serenity-but-the-code-only.patch

```diff
@@ -0,0 +1,26 @@
+From 5c82c8b83862d518889cb6d0c552413748b37f48 Mon Sep 17 00:00:00 2001
+From: circl <circl.lastname@gmail.com>
+Date: Sat, 19 Mar 2022 19:59:28 +0100
+Subject: [PATCH 1/2] malloc.h doesn't exist on Serenity, but the code only
+ checks for macOS and BSDs
+
+---
+ src/Update.cpp | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/Update.cpp b/src/Update.cpp
+index 50ef666..50f4f6b 100644
+--- a/src/Update.cpp
++++ b/src/Update.cpp
+@@ -5,7 +5,7 @@
+ #ifndef WIN
+ #include <sys/param.h>
+ #endif
+-#if !defined(MACOSX) && !defined(BSD)
++#if !(defined(MACOSX) || defined(BSD) || defined(__serenity__))
```

Comment:
Let's stick to the existing style here, patches are not meant to do what _you_ think is best, but should has as few changes as possible.

Reply:
Oh, I thought the original style didn't work for 3 conditions